docs/user/openstack/README.md

```diff
@@ -199,6 +279,53 @@ Wait for the `<cluster name>-api` server comes up and you can make your changes
 Even if the installer times out, the OpenShift cluster should still come up. Once the bootstrapping process is in place, it should all run to completion.
 So you should be able to deploy OpenShift without any floating IP addresses and DNS records and create everything yourself after the cluster is up.
 
+## Installing with Kuryr SDN
+
+To deploy with Kuryr SDN instead of the default OpenShift SDN, you simply need
+to modify the `install-config.yaml` file to include `Kuryr` as the desired
+`networking.networkType` and proceed with the same steps as with the default
+OpenShift SDN:
+
+```yaml
+apiVersion: v1
+...
+networking:
+  networkType: Kuryr
+  ...
+platform:
+  openstack:
+    ...
+    trunkSupport: true
+    octaviaSupport: true
+    ...
+```
+
+
+**NOTE:** both trunkSupport and octaviaSupport are automatically discovered
+by the installer, so there is no need to set them. But if your env don't meet
+both requirements Kuryr SDN will not properly work, as trunks are needed to
+connect the pods to the OpenStack network and Octavia to create the OpenShift
+services.
+
+### Known limitations of installing with Kuryr SDN
+
+There are known limitations when using Kuryr SDN:
+
+   * There is an amphora load balancer VM being deployed per OpenShift svc
+   with the default Octavia load balancer driver (amphora driver). If the
+   environment is resource constrained it could be a problem to create a large
+   amount of services.
+   * Depending on the Octavia OpenStack version, UDP listeners are not
+   supported. This means that OpenShift UDP services are not supported.
+   * There is a known limitation of Octavia not supporting listeners on UDP
+   and TCP on the same port. Thus services expose the same port for UDP and TCP
+   are not supported -- only the TCP listener will be created.
+   * Due to the above UDP limitations of Octavia, Kuryr is forcing pods to
+   use TCP for DNS resolution (`use-vc` option at `resolv.conf`). This may be a
+   problem for pods running go applications compiled with `CGO_DEBUG` flag
+   disabled as that forces to use the `go` resolver that is only using UDP and
+   is not considering the `use-vc` option added by kuryr to the `resolv.conf`.
```

Comment:
This is also an issue for [musl](https://www.musl-libc.org/)-based containers, as it's resolver doesn't support `use-vc` option. This would include e.g. images built from [alpine](https://alpinelinux.org/).